vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -225,6 +225,8 @@ void UwpReactInstance::Start(const std::shared_ptr<IReactInstance>& spThis, cons
     devSettings->useDirectDebugger = settings.UseDirectDebugger;
     devSettings->loggingCallback = std::move(settings.LoggingCallback);
     devSettings->jsExceptionCallback = std::move(settings.JsExceptionCallback);
+    devSettings->useJITCompilation = settings.EnableJITCompilation;
+    devSettings->debugHost = settings.DebugHost;
```

Comment:
It would be better to allow the host name and port to be set independently. The code in ReactWindows does this, but I did not extend this to the devSettings struct at the time. Perhaps we should create an issue to investigate making this change.

Reply:
Created #2593 for this. I agree it's better to separate them out. First time I saw this code I was confused that I needed to include the port. honestly I don't feel that strongly about separating them as much as just making the naming less confusing.